lua/entities/gmod_artron_inhibitor/shared.lua

```diff
@@ -0,0 +1,17 @@
+-- Artron Inhibitor by Nova Astral, an edit of the Time Distortion Generator by parar020100 and JEREDEK
+
+ENT.Type = "anim"
+ENT.Base = "base_gmodentity"
+
+ENT.PrintName = "#TARDIS.ArtronInhibitor"
```

Comment:
There are missing translations for the following strings
- `TARDIS.ArtronInhibitor`
- `TARDIS.ArtronInhibitor.Instructions`
- `ArtronInhibitor.Starting`
- `ArtronInhibitor.Disabled`
- `ArtronInhibitor.Enabled`

These should be added into the `i18n/en.json` file and run the Generate Languages task in VSCode (Ctrl+Shift+P -> Run task -> Generate Languages)

Note you need PowerShell Core installed for this to work

Reply:
Added language but I don't have PowerShell Core and I don't know which one is the correct one.

Reply:
Looks good! Note I've made some small tweaks to match the TDG language